package.json

```diff
@@ -44,6 +44,7 @@
   },
   "dependencies": {
     "config": "^1.26.1",
+    "ioredis": "^3.1.4",
```

Comment:
Not really needed. `node-resque` uses it as it's default library: https://github.com/taskrabbit/node-resque/blob/master/package.json#L28